tests/test_vpc.py

```diff
@@ -592,6 +608,7 @@ def test_multi_attribute_ingress(self):
         manager = p.get_resource_manager()
         self.assertEqual(len(manager.filter_resources(resources)), 1)
 
+    @functional
```

Comment:
this test is not a good functional candidate, its has pre-condition dependencies.